src/measure_theory/measure_space.lean

```diff
@@ -1303,7 +1303,10 @@ add_decl_doc volume
 section measure_space
 variables {α : Type*} {ι : Type*} [measure_space α] {s₁ s₂ : set α}
 
-notation `∀ᵐ` binders `, ` r:(scoped P, volume.ae.eventually P) := r
+notation `∀ᵐ` binders `, ` r:(scoped P, filter.eventually P (measure.ae volume)) := r
+
+/-- The tactic `exact volume`, to be used in optional arguments. -/
+meta def volume_tac : tactic unit := `[exact volume]
```

Comment:
This will only work if the `measure_theory.measure_space` namespace is open at the time the tactic is called. Is there a reason to use an autoparam instead of giving `measure` as a default value?

Reply:
`:= volume` in `def integrable` doesn't work because at this moment `α` has no `measure_space` instance, only `measurable_space`.

Reply:
Aha. Those are too similar for me to tell the difference when I'm just glancing through!